### PR TITLE
DataGrid: Fix code snippets in onEditorPreparing

### PR DESCRIPTION
**Pull request by RomanTsukanov**
Fixes docissues [T893002](https://supportcenter.devexpress.com/internal/ticket/details/T893002) and [T893004](https://supportcenter.devexpress.com/internal/ticket/details/T893004)

### CODE REVIEW

File: api-reference/10 UI Widgets/dxFilterBuilder/1 Configuration/onEditorPreparing.md [hunk 1]
```diff
@@ -94,11 +95,12 @@ In the following code, a default editor is replaced with the DevExtreme [TextAre
     export class AppComponent {
         onEditorPreparing (e) { 
             if (e.dataField == "description") {
+                const defaultHandler = e.editorOptions.onValueChanged;
                 e.editorName = "dxTextArea"; 
                 e.editorOptions.showClearButton = true;
                 e.editorOptions.onValueChanged = (event) => {
                     let value = event.value;
-                    e.setValue(value.toLowerCase()); 
+                    defaultHandler(value.toLowerCase());
```
Comment (onebalaban) [inline api-reference/10 UI Widgets/dxFilterBuilder/1 Configuration/onEditorPreparing.md]: It seems that this code doesn't work: https://codepen.io/romanbalaban/pen/PoZdxWE

Comment (RomanTsukanov) [reply api-reference/10 UI Widgets/dxFilterBuilder/1 Configuration/onEditorPreparing.md]: @vconst

Comment (vconst) [reply api-reference/10 UI Widgets/dxFilterBuilder/1 Configuration/onEditorPreparing.md]: defaultHandler(event)

**Comment by RomanTsukanov** [comment]
@onebalaban It turns out that in some cases `setValue` is preferable over the default handler. I've updated the code snippets and tried to clarify which API should be use in which case.

**Review by vconst — APPROVED** [review]

**Pull request merged**